crates/re_types_builder/src/codegen/python.rs

```diff
@@ -218,6 +347,14 @@ impl PythonCodeGenerator {
                 0,
             );
 
+            if ext_class.found {
+                code.push_unindented_text(
+                    format!("from .{} import {}", ext_class.module_name, ext_class.name,),
+                    1,
+                );
+            }
```

Comment:
```suggestion
            } else {
                code.push_str(&format!("# You can extend this class by creating a {}\n", ext_class.file_path));
            }
```

Reply:
actually, that should probably go in the top of the file after the "DO NOT EDIT" warning